src/APICore.ts

```diff
@@ -129,13 +130,20 @@ export default class API {
             ...args,
             headers: {
                 Authorization: `Bearer ${this.accessToken}`,
-                Accept: 'application/json',
                 ...(args.headers || {}),
             },
         };
 
-        const response = await fetch(this.getUrlFromRoute(route), init);
-        return handleResponse<T>(response, this.handlers);
+        const req = async () => {
+            const response = await fetch(this.getUrlFromRoute(route), init);
+            if (this.isThrottled(response.status)) {
+                throw response;
+            }
+            return response;
+        };
+
+        const out = await backOff(req, {retry: (e: Response) => this.isThrottled(e.status)});
```

Comment:
The reason we need to `throw` on 429, and then retry on 429 (so... two times the verification), is because we don't want to retry for any type of errors, just from those that we know we can potentially recover.